networking/p2p/src/test/java/tech/pegasys/artemis/networking/p2p/jvmlibp2p/gossip/AggregateTopicHandlerTest.java

```diff
@@ -70,4 +74,18 @@ public void onNewAggregate() {
     assertThat(topicCaptor.getAllValues().size()).isEqualTo(1);
     assertThat(topicCaptor.getValue()).isEqualTo(topicHandler.getTopic());
   }
+
+  @Test
+  public void accept_invalidAttestation_badState() throws Exception {
+    final AggregateAndProof aggregate = DataStructureUtil.randomAggregateAndProof(1);
+    Store.Transaction transaction = storageClient.getStore().startTransaction();
+    transaction.putBlockState(aggregate.getAggregate().getData().getBeacon_block_root(), null);
```

Comment:
nit - I don't think you need to explicitly set the block state to null since the attestation is random

Reply:
done